os/arch/arm/src/amebasmart/amebasmart_serial.c

```diff
@@ -1351,12 +1242,11 @@ void up_serialinit(void)
 #endif
 
 #ifdef CONFIG_PM
-	/* Register to receive power management callbacks */
-	ret = pm_register(&g_serialpm.pm_cb);
+	/* Domain registration for operation control */
+	(void)pm_domain_register("UART");
+	(void)pm_domain_register("LOGUART");
```

Comment:
What is difference between UART and LOGUART? and what is the usage of them?

Reply:
LOGUART is console uart, UART is MICOM uart. In tash_command.c, there is a domain suspend operation. Thus as mentioned by Eunwoo. I will keep just UART.
The usage for UART will be to suspend/resume domain when there is communication/operation with MICOM.